src/lib/viewers/controls/annotations/AnnotationsControls.tsx

```diff
@@ -62,18 +65,20 @@ export default function AnnotationsControls({
         return null;
     }
 
+    const isDrawingActive = annotationMode === AnnotationMode.DRAWING;
+
     return (
         <div className="bp-AnnotationsControls">
             <AnnotationsButton
                 data-resin-target="draw"
                 data-testid="bp-AnnotationsControls-drawBtn"
-                isActive={annotationMode === AnnotationMode.DRAWING}
+                isActive={isDrawingActive}
                 isEnabled={showDrawing}
                 mode={AnnotationMode.DRAWING}
                 onClick={handleModeClick}
                 title={__('drawing_comment')}
             >
-                <IconDrawing24 />
+                <IconDrawing24 style={{ fill: isDrawingActive ? annotationColor : '#fff' }} />
```

Comment:
Maybe this can be moved to a constant? It seems a little hardcoded here.

Reply:
Why does this icon have different behavior and visual treatment from the others?

Reply:
In the designs, it looks like whenever the drawing button is active, the fill color matches the color in the ColorPickerControl swatch.

 
![Screen Shot 2020-12-04 at 4 18 03 PM](https://user-images.githubusercontent.com/7213887/101227344-5d8f0300-364c-11eb-8eec-442368a1ac41.png)

Reply:
Another reason is that with the introduction of drawing annotations, there is an added context of a color that the user has set (whereas region / highlight annotations are color agnostic).

Reply:
Is it just me or is there an extra white border on the left of the active state of the drawing button above?

Reply:
I think that might be a bug in zeplin, in this new screenshot I no longer see it.
![Screen Shot 2020-12-07 at 1 00 07 PM](https://user-images.githubusercontent.com/7213887/101404932-2b6be400-388c-11eb-8944-10a337e3c149.png)

Reply:
Should we move this logic to upper level so it will become `<IconDrawing24 style={{ fill: annotationColor }} />`

Reply:
Could you clarify a bit about what you mean by upper level? Do you mean instead of the ternary inside of the style prop, I do a ternary with the icon?

`isDrawingActive ? <IconDrawing24 style={{ fill: annotationColor }} /> : <IconDrawing24 style={{ fill: '#fff' }} />`

Reply:
Can `IconDrawing` accept `fill` as a prop instead of using inline styles?

Reply:
Yep that works.